### ISSUE TITLE
WHAT discussion: What are the knowledge gaps

### ISSUE THREAD

**Issue by crossley** [crossley/ARCANI#2]
Please share your feedback on this group discussion by replying in this thread. Your contributions are important as we aim to identify gaps in knowledge, pinpoint collaborators who can help us bridge these gaps, and explore how sustainable collaboration within the ARCANI network can address these issues. We will incorporate the thoughts and opinions expressed here into a white paper focused on these critical questions.

**Comment by gmfricke** [comment]
How do we remove hallucinations? Is removing all hallucinations desirable (to Bruno's talk)

**Comment by jarmarshall** [comment]
How do we reduce training and compute requirements for AI models to be more comparable to animal levels of experience

**Comment by ABBarron** [comment]
animal brains don't start from de-novo.  How do instincts evolve and how do instincts scaffold learning?

**Comment by jarmarshall** [comment]
what learning theory do we have that's animal-like, but more sophisticated than Rescorla-Wagner?

**Comment by ABBarron** [comment]
Backpropogation through time is biologically impossible. And recurrent ANNs are really hard to train. But animal brains seem to solve this. And I don't know how.  It keeps me awake at night.

**Comment by jarmarshall** [comment]
Do we even need a brain to learn, e.g. slime mould

**Comment by dracothraxus** [comment]
•	Terminology and semantics fuzzy and varied across the fields
•	Robotics undergoing transformation – modern AI stuff driven
•	Embodiment affects cognition
•	Mechanical bio-inspiration
•	How do we assess progress in both fields? Generality versus specificity? Task-specific versus general?

**Comment by ookwrd** [comment]
How do models and representations constrain cognition in humans, animal, or any diverse intelligences?

**Comment by dracothraxus** [comment]
Do modern AI + robot developments need causality understanding, to what, and if so, to what extent?

**Comment by ABBarron** [comment]
> what learning theory do we have that's animal-like, but more sophisticated than Rescorla-Wagner?

yeah - and it worries me that all nodes in ANNs are identical and operate this loosely biological learning rule. Neurons are not identical!

**Comment by beebee44** [comment]
how does cognition intervene to generate behavioural control

**Comment by jarmarshall** [comment]
brains evolved to solve movement - cognition scaffolds on top of this, re-using the same circuits?

**Comment by dracothraxus** [comment]
What is the benefit of biomorphic?

**Comment by RMenary** [comment]
> brains evolved to solve movement - cognition scaffolds on top of this, re-using the same circuits?

Agreed, the recruitment of sensors-motor capacities for symbolic cognition is not well understood.

**Comment by dracothraxus** [comment]
It's impossible to be impossible @ABBarron

**Comment by ookwrd** [comment]
The Universal Cognitive Translator: do we have the tools to reverse engineer cognitive representations across species, to recreate the cognitive (e.g. intuitive physics) or phenomenology (e.g. what it feels like to be a fruit fly) of one into another? For an example, can we recreate the sense of physics that a (particular) chimp has and create a virtual simulation of it?

**Comment by dracothraxus** [comment]
How do we collect evidence that the systems we create or study have the capabilities that we ascribe to them?

And what is the burden of evidence in different fields and contexts?

**Comment by dracothraxus** [comment]
Is it a conscious decision or a reflex?

**Comment by ookwrd** [comment]
Can we go beyond teaching human language to animals, and creating interfaces for animals to speak in their own « language » and another species species listening in hearing it in their own « language ». (Note: Language scare-quoted here.)

**Comment by gmfricke** [comment]
There is a serious gap in goal alignment between machine learning engineers who generally develop algorithms to solve a very narrow problem. There is largely no awareness in the machine learning algorithm literature in neuroscience or cognitive science. A basic education in neuroscience among ML developers and ML among cognitive scientists would perhaps be beneficial - and addressable through collaborative networking. ML engineers are large focused on how to speed up Amazon transactions, not implications for general AI.

**Comment by jarmarshall** [comment]
how do we find new learning algorithms from the brain, rather than impose our current algorithms on our current understanding of the brain, e.g. seeing support vector machines in insect antennal lobe

**Comment by RMenary** [comment]
Gap: The link between sensory-motor intelligence and symbolic - verbal and quantitative - thinking.

**Comment by crossley** [comment]
To what extent are biological system capable of interleaving new behaviours with old, without suffering catastrophic interference, and can these mechanisms imbue artificial systems with similar traits. This would seem to become even more essential as artificial systems are placed in long-term continual learning roles.

Some bioligcal systems are very good at processing and making decisions form visual information from complex scenes with cheap sensors. How? And can these mechanisms be further mapped to artifical systems for cheaper more effective systems. 

A big question / cornerstone is making good artificial systems that are also efficient.

**Comment by fgashby** [comment]
Humans have multiple functionally and anatomically distinct networks that can learn in qualitatively different ways. This gives humans a flexibility advantage over many AI systems. One important knowledge gap about human learning is how these various learning systems coordinate, how the knowledge they acquire is integrated, and how control is passed from one system to another.

The human brain does not rely on backpropagation. One way it overcomes this limitation is to use a variety of qualitatively different learning algorithms that are each ideally suited for a different type of learning. For example, a prominent proposal that was popularized by Kenji Doya is that synaptic plasticity follows Hebbian learning rules  in cortex, reinforcement learning rules in the basal ganglia, and supervised learning rules in the cerebellum.

**Comment by dracothraxus** [comment]
@gmfricke there are initiatives like ELLIS in Europe that try to remedy this - we could potentially model part of ARCANI on some of that

**Comment by ABBarron** [comment]
> There is a serious gap in goal alignment between machine learning engineers who generally develop algorithms to solve a very narrow problem. There is largely no awareness in the machine learning algorithm literature in neuroscience or cognitive science. A basic education in neuroscience among ML developers and ML among cognitive scientists would perhaps be beneficial - and addressable through collaborative networking.

totally! And on the flipside I'm embarrassed how little I know about and understand LMMs

**Comment by ookwrd** [comment]
Do we have an agreement on a framework for models that extract and map goals from various species (natural or artificial)?

**Comment by dracothraxus** [comment]
Accessibility to understanding behaviour e.g. hallucinations in artifical and natural systems

**Comment by dracothraxus** [comment]
Do we hold AI to higher standards, risk reward, how do we approach that

**Comment by crossley** [comment]
Biological systems are better substrate inspiration for ensembles of experts / AI as layers of systems. Similar idea to Greg's comment above about multiple learning and memory systems.

**Comment by dracothraxus** [comment]
Transparency and new tools and standards are likely needed for the "new" AI - can the network help this?

**Comment by stenti** [comment]
Thoughts from our table:
How do we get from basal congnition to high order cognition?
Is navigation the first sign of basal intelligence?
There are so many definitions of intelligence. How can we pull these concepts of different types of inteligence across to artificial inteligence eg creative intelligence. Generalised testing doesn't work for humans and cant compare inteligence across cultures let alone animal inteligence, why would it work for AIs.
How are we definining intelligence across fields? Should we expand the definition of integligence beyond general inteligence. 
Frustration: Expecting animals to learn using human language or human-centric tasks - enforcing a non-natural learning - we are limiting ourselves this way
Artificial inteligence models can act as if a specific species - speed up the process of exploring data we already have - bypass some ethical issues - however AI in this way cannot addapt to changes - gap in knowledge is the ability to learn novelly without specific training?
Deep learning models are clearly not how animals are solving the problem - are we being limited in our exploration by the tools we already have?

**Comment by crossley** [comment]
Knowledge gap: How multiple systems / modules are coordinated and controlled in learning and behaviour.

**Comment by dracothraxus** [comment]
Sustainability perspective - insects?

**Comment by ookwrd** [comment]
Is the Internet of Animals back yet? Can we make it VR-augmented this time? (DI had a nice grant project on this.)

**Comment by ABBarron** [comment]
> To what extent are biological system capable of interleaving new behaviours with old, without suffering catastrophic interference, and can these mechanisms imbue artificial systems with similar traits. This would seem to become even more essential as artificial systems are placed in long-term continual learning roles.
> 
> Some bioligcal systems are very good at processing and making decisions form visual information from complex scenes with cheap sensors. How? And can these mechanisms be further mapped to artifical systems for cheaper more effective systems.
> 
> A big question / cornerstone is making good artificial systems that are also efficient.

How is it that animals can avoid catastrophic forgetting?

**Comment by ookwrd** [comment]
> How do we remove hallucinations? Is removing all hallucinations desirable (to Bruno's talk)

I love this, especially in the light of Sam’s talk on what I’d call « cognitive noise-canceling ». Let’s call them confabulation though please? Hallucinations would be something else :)

**Comment by stenti** [comment]
Thoughts from our table 2:
Difficulty in implementing the models we build in existing hardware - are we being limited by the technology we have - neuromorphic tech may be a solution - however it is still very far removed from any real brains
brains bodies and behaviours have coevolved within an environment - important to look at inteligence within all of these 
Deep learning black box AI can do amazing things but is unintelliagable,  biomimetic AI more of an open box but currently less capable

**Comment by dracothraxus** [comment]
Don't be too functionally-focused - modelling animals / bio for serendipitious, unpredictable discoveries *ducks and hides from some risk-averse grant funding agencies*

**Comment by jarmarshall** [comment]
embodied cognition - don't try and solve everything with the brain, solve with the body as well

**Comment by jarmarshall** [comment]
☝️ Is it even possible to be intelligence if you don't have a body?

**Comment by ABBarron** [comment]
> > How do we remove hallucinations? Is removing all hallucinations desirable (to Bruno's talk)
> 
> I love this, especially in the light of Sam’s talk on what I’d call « cognitive noise-canceling ». Let’s call them confabulation though please? Hallucinations would be something else :)

Agreed! But it's interesting we call a weird thing a LLM does a "hallucination". That says something about how we perceive them.

**Comment by crossley** [comment]
Hierarchical system control engineering or other as inspirations for research programs that might shed light on the mutliple system learning and control problem.

**Comment by dracothraxus** [comment]
Geographical and cultural diversity in the network of people shaping this - ARCANI to play a role in diversifying?

**Comment by beebee44** [comment]
AI commonly focuses on cognition/intelligence but the selection problem suggests that important constraints are generated by motivational/emotional concerns. Some consideration of this issue and involvement of groups interested in these functions would be important.

**Comment by ABBarron** [comment]
> What is the benefit of biomorphic?

good question! learning speed? operating costs?  We think of biological brains a metabolically expensive but they are cheap and efficient compared to a LLM

**Comment by stenti** [comment]
Thoughts from our table 3:
Hacking the mind, parasitizing a system in order to change its behaviour towards its own goal - fungi do this to ants - this is a risk as AI develops - can you hypnotize an ant? can you hypnotize a machine? The barrier is that we dont understand how to hijak cognitive processes. Creativly breaking the system in a consistent way can give you insight into a system. Generally encoded knowlege - need a framework to extract and compare mathmatics acrss representation not just humans. (needs revision)

**Comment by crossley** [comment]
> AI commonly focuses on cognition/intelligence but the selection problem suggests that important constraints are generated by motivational/emotional concerns. Some consideration of this issue and involvement of groups interested in these functions would be important.

Spot on!

**Comment by jarmarshall** [comment]
Chris Reid: many mammals can learn to walk within a few hours of birth, others take years. What is it about the brains/innate structures in these mammals, and can we take principles from these to enhance learning speed in artificial models/bodies?

**Comment by dracothraxus** [comment]
and what is the meaning of / significance of time and learning in artificial and natural contexts (especially with federated, offline high throughput learning)

**Comment by ABBarron** [comment]
> and what is the meaning of / significance of time and learning in artificial and natural contexts (especially with federated, offline high throughput learning)

what's federated offline high throughput learning?

**Comment by ookwrd** [comment]
Can we create universal modifiers that work the same way in several (3 or more) cognitive substrates?

**Comment by jarmarshall** [comment]
How important are models of the environment, including self, in improving learning rate?

**Comment by ABBarron** [comment]
> How important are models of the environment, including self, in improving learning rate?

Yes!!!!!! And in animals what are those models like, where do they come from, how do they develop and evolve? Is "model" even the right metaphore for an animal or would Barbara Webb take exception?

**Comment by stenti** [comment]
fundametal flaw in artificial inteligences is that they have no motivation that we havent given them. All living organisms are motivated somewhat in ensuring survival - how can we give this type of pulls for artificial inteligences? (we have all seen that movie)
There is also the issue of what happens at failure - is there a mechanism of self punishment
Can you have any inteligence without a body?

**Comment by ookwrd** [comment]
Can we push funding sources in the opposite direction to fear-based projects at least part of the time (e.g. existential risk, etc.)?

**Comment by dracothraxus** [comment]
TABLE mid summaries of past table discussions:

- interesting points summary: some mammals can learn very quickly, some slowly, can we learn from these to apply to AI?
- long journey of learning from sensory-motor to more advanced - what can we learn from that for higher order thinking - gap?
- huge disconnect between the neuroscience and machine learning, don't care, don't know?
- if the brain is a predictive machine, what does a model of the world look like? In silico observable, in nature harder to observe, and parallels with machine learning opaqueness
- why the AI can't perform a similar task in a different context? Generality limitations, possible solutions from layered approaches? bio-inspired?
- modelling bio-inspired systems for serendipitious discovery

**Comment by stenti** [comment]
Is navigation the first sign of inteligence - directed movement with some goal. Is this how we could identify inteligence?

**Comment by stenti** [comment]
> Is navigation the first sign of inteligence - directed movement with some goal. Is this how we could identify inteligence?

Requires a body not just a mind

**Comment by ABBarron** [comment]
> Knowledge gap: How multiple systems / modules are coordinated and controlled in learning and behaviour.

yes! brains have that network of networks thingo feature.  How do we wrestle with that?

**Comment by crossley** [comment]
What is intelligence in the first place? We need a shared language that spans our lightly connected fields. This is a huge gap that seems quite essential to address. This might be the semantic hygiene idea.

**Comment by stenti** [comment]
If different fields of research use different language and semantics are we able to communicate and collaborate effectivly? Can we solve this with a LLM?

**Comment by dracothraxus** [comment]
terminology commonality and understanding emerging again as a key issue especially in diverse networks

**Comment by dracothraxus** [comment]
@ABBarron federated high throughput learning is millions of robots in kitchens all around the world sharing their learnings and also running 100x realtime speed simulations in parallel.

e.g. "I accidentally stabbed my master Andrew - he appears to be unhealthy - suggest other robots not do this"

**Comment by MGGuiraud** [comment]
A limitation is the population of researchers working both in neurosciences (biological) + AI/engineering. Need dual background to overdose current obstacles in the field -> we need funding for these programs (e.g.: dual Bachelor/master degrees), there is less funding on the side of neuroscience.

**Comment by patrickmcgivern** [comment]
Challenge of bridging between different disciplines given the complexity of the problems/frameworks within separate fields, distinct methodologies, and the opaqueness of central concepts associated with intelligence

**Comment by MGGuiraud** [comment]
Perceived risks versus real risks of AI - not going to destroy the world. Yet, we already do not mitigate all the risks about data management, AI related to health management, etc.

**Comment by stenti** [comment]
> A limitation is the population of researchers working both in neurosciences (biological) + AI/engineering. Need dual background to overdose current obstacles in the field -> we need funding for these programs (e.g.: dual Bachelor/master degrees), there is less funding on the side of neuroscience.

Funding for skill sharing between research groups would also be great further up the chain

**Comment by MGGuiraud** [comment]
For neuromorphic AI we need to understand more about the neuroscience, how the different structures communicate between each others rather than making wild assumptions about blobs (cortex...).

**Comment by ABBarron** [comment]
> For neuromorphic AI we need to understand more about the neuroscience, how the different structures communicate between each others rather than making wild assumptions about blobs (cortex...).

yes absolutely! A brain is not an undifferentiated network! Far from it! And it's not made of blobs either!

**Comment by crossley** [comment]
Here we go.... what is consciousness, why did it evolve, what does it add if anything the system. Do we need and want our artificial systems to have it? Whoo.

**Comment by Somayeh-h** [comment]
There’s a lot of knowledge in various neuroscience related topics, however, there is still not a lot of collaborations between these fields. The experts in these fields usually have different goals, and what makes collaborations more effective is to be able to ask the kind of questions, and have the kind of discussions that results into finding new research gaps. 
Sometimes the finding the right literature is hard to find because the same concepts are phrased and termed very differently - answer might be to use AI based tools to help.

**Comment by dracothraxus** [comment]
Ethics around AI decision making?

**Comment by MGGuiraud** [comment]
Biggest impact questions utilising this network : 1/ plateform for different species with experiments (e.g. https://www.phylopsy.org/ or mimosa?) to understand better cognition and make a basis for consciousness.

**Comment by ookwrd** [comment]
Sharing-co-reviewing research questions and methodologies in diverse cognition research through platforms similar to Phylopsy, Mimosa, etc.

**Comment by ABBarron** [comment]
> Here we go.... what is consciousness, why did it evolve, what does it add if anything the system. Do we need and want our artificial systems to have it? Whoo.

Whoo indeed!

**Comment by HoverflyLab** [comment]
Cultural effects on perceived risks as well as what tasks need to be solved. At the moment it appears to be very US centric, but a different cultural lens could have a strong effect in the questions asked.

**Comment by dracothraxus** [comment]
Cancer to the Cambrian explosion - what can we learn?

**Comment by MGGuiraud** [comment]
How to create a big enough question to cover the whole of the network? What about creating sub-network between people interested by similar topics (ex. Active vision Karin/James/Marie/Andrew ; communication Richard/Olaf…. Etc). Sub-questions: How does active learning works in biological/artificial systems ?

**Comment by crossley** [comment]
There may be some important conversations / gaps to address surrounding various ethical issues of AI development and placement in human societies.

**Comment by dracothraxus** [comment]
IQ test Stanford - what we can learn?

**Comment by HoverflyLab** [comment]
The sustainability of the field: Insects are able to perform amazingly with very limited resources, but robotics and AI are not really appearing to take this into account in the development.

**Comment by MGGuiraud** [comment]
How does the brain simplify perception?

**Comment by MGGuiraud** [comment]
> Biggest impact questions utilising this network : 1/ plateform for different species with experiments (e.g. https://www.phylopsy.org/ or mimosa?) to understand better cognition and make a basis for consciousness.

Sharing methodology!

**Comment by dracothraxus** [comment]
How do we move beyond capitalist considerations?

**Comment by MGGuiraud** [comment]
> A limitation is the population of researchers working both in neurosciences (biological) + AI/engineering. Need dual background to overdose current obstacles in the field -> we need funding for these programs (e.g.: dual Bachelor/master degrees), there is less funding on the side of neuroscience.

Summerschools with hands-on workshop where neuroscientists learn some computing skills and vice versa

**Comment by HoverflyLab** [comment]
Skill uplift needed, need training in both machine learning and neuroscience/behavior.

**Comment by ABBarron** [comment]
> Skill uplift needed, need training in both machine learning and neuroscience/behavior.

yes! a skills-based summer school? Not just for grads - for ECRs and oldies like me!

**Comment by crossley** [comment]
> what learning theory do we have that's animal-like, but more sophisticated than Rescorla-Wagner?

Cool question. The contextual / bayesian inference models from Sam Gershman and Yael Niv come to mind as the next generation of associative learning mathematical models. Anne Collins also has some nice work where she places the Bayesian mathematical models into the basal ganglia circuit. Myself, Greg Ashby (also in attendance), Michael Frank, Kevin Gurney etc have done a lot of work extending models of action selection in basal ganglia circuits to the extent that I think they are importantly different from RW math models.

**Comment by HoverflyLab** [comment]
> > Skill uplift needed, need training in both machine learning and neuroscience/behavior.
> 
> yes! a skills-based summer school? Not just for grads - for ECRs and oldies like me!

Summer schools are a great idea.

**Comment by dracothraxus** [comment]
"what does everyone in the room need to do the things they really want to do that they can't currently do?"

**Comment by gmfricke** [comment]
Can AI's effectively learn appropriate ethics through reinforcement or do they need rules.

**Comment by alexjgillett1984** [comment]
A central database for measures of cognitive domains and intelligence is required. For instance, Nora Newcombe and colleagues has recently pointed out that spatial cognition measures have a number of issues, and that a central database for assessing and refining these is required. A similar point could be made for many cognitive constructs. And this will be a major gap in trying to refine and test artificial intelligence

**Comment by kozzy97** [comment]
Gap in methodology is the measurement problem - how do we operationalise concepts such as memory capacity, attention, intuitive physics, intelligence and then build instruments to test them? How can we improve our measurement tools? This seems crucial if we want to commensurate cognition in diverse species and AI (and appraise the capabilities and consequently the safety of AI systems)

**Comment by philosophyalex** [comment]
One thing that is limiting progress in AI is that it is not surprised by its mistakes, like animals are. It is not embodied, and doesn’t have a dopaminergic system.

**Comment by Daspraelon** [comment]
“Introspection” doesn’t map well to cogsci. On the other hand, “metacognition” can be widely used and is sometimes itself contentious. What does it mean to “stand outside oneself” in practical contexts. (I’d lean in to the difference between asking it this way and getting involved in the endless semantic bickering that arise in-field over metacognition.)

**Comment by HoverflyLab** [comment]
There's a benefit in meeting face to face, but online meetings work when social connections have already been made (and saves travelling time and time away from family and other commitments)
